src/plugins/dotplot/src/DotPlotWidget.cpp

```diff
@@ -778,7 +779,16 @@ bool DotPlotWidget::sl_showSettingsDialog(bool disableLoad) {
 
 // ask user if he wants to save dotplot first
 void DotPlotWidget::sl_showDeleteDialog() {
```

Comment:
Add an explicit parameter about the mode: show or not to show 'cancel' button: isCancelable = true